ios/keyman/Keyman/Keyman/AppDelegate.swift

```diff
@@ -19,6 +20,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
   func application(_ application: UIApplication,
                    didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil) -> Bool {
+    Manager.applicationGroupIdentifier = "group.KM4I"
+    #if DEBUG
+      Manager.shared.isDebugPrintingOn = true
+    #endif
+    Manager.shared.openURL = UIApplication.shared.openURL
+
```

Comment:
What happened to the `setCanRemoveDefaultKeyboard` component from the original KMEI-side implementation?

Also, I recall you mentioning about moving certain components out of the engine and into the app - I take it the AppDelegate class was one such part, hence blending the two here?

Reply:
`setCanRemoveDefaultKeyboard` is now the `canRemoveDefaultKeyboard` property in `Manager`.

The KeymanEngine project builds a demo app for testing. I moved out all components which were only part of that demo app out into a separate target so that it is not bundled into the framework. This includes `KeymanEngineDemo/AppDelegate.swift` but does not affect `Keyman/AppDelegate.swift` (since the Keyman app was always a separate target).

The addition here is moved from `Keyman/MainViewController.swift`. It was moved because we need to set `Manager.applicationGroupIdentifier` before `Manager.shared` is used due to set up work which depends on the `applicationGroupIdentifier`. I believe in the existing ObjC code, sometimes `KMManager` would use an unset `applicationGroupIdentifier` since `KMManager.sharedInstance()` was called first when the views are being initialized.